app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -535,28 +536,41 @@ function handleImageUpload(event, randomIdNumber) {
   }
 }
 
+function toggleItemSummaryHtml(item) {
```

Comment:
nitpick (not blocking) - it's a tiny thing, but this function name with 'toggle' says to me that the function will collapse/expand the details/summary panel. Maybe something like `updateItemSummaryHtml` would better reflect what's happening here 🤔 

Feel free to disregard though

Reply:
Done